examples/source/1.components/amp-autocomplete-email.html

```diff
@@ -81,8 +81,12 @@ <h2>Basic usage</h2>
 
     Note: Because server-side filtering is mandatory in the email format, it is wise to pair the `src` and `query` attributes
     in order to passed user inputs to a statically generated endpoint. For example if a `src="https://example.com` and `query="q"`, then a user who types in `abc` will get the fetch JSON result from `https://example.com?q=abc`.
+
+    Note: that the `data-amp-autocomplete-opt-in` attribute has
+    to be included to amp4email amp documents
     -->
     <amp-autocomplete
+      data-amp-autocomplete-opt-in
```

Comment:
Sorry, should have been more clear - using `data-amp-autocomplete-opt-in` is not valid AMP - users should not be the ones to include it.
Rather, email clients inject it into their email environment when displaying/sending the email, after the document has passed the AMP validator. Is it possible to replicate that behavior in the displayed document in the playground without it being part of the manual code sample?

Reply:
my mistake - I should have checked the source - thank you @caroqliu! I updated the code to add it at runtime, and added @sebastianbenz to review the method of adding it.